lib/global_config.js

```diff
@@ -157,3 +158,44 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
+};
+
+/**
+ * Return default configuration for XmlParser.
+ *
+ * @returns {object}
+ */
+exports.getXmlParserConfiguration = function ()
+{
+  if (xmlParserConfiguration === undefined || xmlParserConfiguration  === null) {
+    return defaultXmlParserConfiguration;
+  }
+  return xmlParserConfiguration;
+}
+
+/**
+ * Set custom xmlParser configuration or use default.
+ * Parametrs that you can override:
+ *   ignoreAttributes - default true,
```

Comment:
maybe let's add more info here what are those parameters for and what will be the impact of changing them